h/tasks/indexer.py

```diff
@@ -46,9 +60,22 @@ def reindex_user_annotations(userid):
     if errored:
         log.warning('Failed to re-index annotations %s', errored)
 
+    if celery.request.feature('index_es6'):
+        indexer = BatchIndexer(celery.request.db, celery.request.es6, celery.request)
+        errored = indexer.index(ids)
+        if errored:
+            log.warning('Failed to re-index annotations into ES6 %s', errored)
+
 
 def _current_reindex_new_name(request):
     settings = celery.request.find_service(name='settings')
-    new_index = settings.get(SETTING_NEW_INDEX)
+    new_index = settings.get('reindex.new_index')
+
+    return new_index
+
+
+def _current_reindex_new_es6_name(request):
+    settings = celery.request.find_service(name='settings')
+    new_index = settings.get('reindex.new_es6_index')
```

Comment:
I think that's prob a good idea in this particular case. I considered doing that for all the copy paste's but then I figured we'd just delete them anyway and then we'd have a function that's only called once so that would be kinda silly.